ui/widgets/resizable.js

```diff
@@ -131,11 +132,16 @@ $.widget( "ui.resizable", $.ui.mouse, {
 
 			this.elementIsWrapper = true;
 
+			marginTop = this.originalElement.css( "marginTop" );
+			marginRight = this.originalElement.css( "marginRight" );
+			marginBottom = this.originalElement.css( "marginBottom" );
+			marginLeft = this.originalElement.css( "marginLeft" );
+
 			this.element.css( {
-				marginLeft: this.originalElement.css( "marginLeft" ),
-				marginTop: this.originalElement.css( "marginTop" ),
-				marginRight: this.originalElement.css( "marginRight" ),
-				marginBottom: this.originalElement.css( "marginBottom" )
+				marginLeft: marginLeft,
+				marginTop: marginTop,
+				marginRight: marginRight,
+				marginBottom: marginBottom
 			} );
 			this.originalElement.css( {
```

Comment:
Setting (as opposed to getting) margins with a shortcut is usually safe, could this just be `{ margin: 0 }`?